python/lsst/sphgeom/_healpixPixelization.py

```diff
@@ -0,0 +1,174 @@
+# This file is part of sphgeom.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (http://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+__all__ = ["HealpixPixelization"]
+
+import numpy as np
+import healpy as hp
+
+from ._sphgeom import RangeSet, Region, UnitVector3d
+from ._sphgeom import Box, Circle, ConvexPolygon, Ellipse
+from .pixelization_abc import PixelizationABC
+
+
+class HealpixPixelization:
```

Comment:
```suggestion
class HealpixPixelization(PixelizationABC):
```
Since this is a pure-Python class, you can inherit from `PixelizationABC` the usual way.  It's only the C++ ones that need to use `register`.